README.md

```diff
@@ -1,19 +1,41 @@
-# React render() Must Not Have Side Effects
+# React render() must not have side effects
 
-## Objectives
+???
 
-1. Review the requirements of `render()`
-2. Explain why `render()` must not have side effects
+# React `render()` Quiz
 
-## Overview
+?: `render()` is the only required method in a React component.
 
-This quiz is meant to give students a chance to review how to write a proper
-`render()` function. It's a review of pure functions and also a refresher about
-`render()` before we dive into props and state.
+[X] True.
+[ ] False.
 
-We can use this space to go over anything we might have missed in earlier units,
-and to really drive home how important `render()` is.
+?: `render()` can only return elements, not other React components.
 
-## Resources
+[ ] True.
+[X] False.
+
+?: `render()` must be a pure function.
+
+[X] True.
+[ ] False.
+
+?: `render()` can return multiple child elements.
```

Comment:
It is possible just to provide some sample code, and then ask them to indicate (true/false) if it's valid? That might remove the ambiguity.